src/window.rs

```diff
@@ -171,13 +171,14 @@ impl WindowBuilder {
 
     pub fn build(self) -> Window {
         let event_loop = winit::event_loop::EventLoop::new();
-        let mut builder = winit::window::WindowBuilder::new();
+        let mut builder = winit::window::WindowBuilder::new()
+            .with_min_inner_size(winit::dpi::Size::Logical((64, 64).into()));
         if let Some(title) = self.title {
             builder = builder.with_title(title);
         }
         if let Some(size) = self.size {
             builder = builder
-                .with_inner_size(winit::dpi::Size::Physical((size.width, size.height).into()));
+                .with_inner_size(winit::dpi::Size::Logical((size.width, size.height).into()));
```

Comment:
This seems undesired. If I'm creating a window that I want to render 800x600 into, I don't expect 1600x1200 to be created (because of the scale factor).
I think the logical sizes make more sense for UI applications, while game applications still only care about physical.

Reply:
Bevy uses logical size, for example.

https://github.com/bevyengine/bevy/blob/main/crates/bevy_winit/src/winit_windows.rs#L56

Reply:
weird. If you have more information about this, I'd like to understand.
Maybe the end user doesn't care what the exact pixels are...

Reply:
We can look at this from a user perspective. With current behavior, if I will run an example on iMac 5k, I will get a tiny window (and it would be hard to see content inside it). But if I will try to run the example on a low-end mobile device (with HD display), it can be half of its screen size.